### PR TITLE
JBTM-2686 Update to add a Heuristic example in the karaf osgi-jta quickstart

### PR DESCRIPTION
**Pull request by zhfeng**
https://issues.jboss.org/browse/JBTM-2686

### CODE REVIEW

File: karaf/osgi-jta/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Build and Run the Quickstart
         karaf@root()> repo-add mvn:org.ops4j.pax.jdbc/pax-jdbc-features/0.8.0/xml/features
         karaf@root()> feature:install pax-jdbc-pool-narayana jdbc pax-jdbc-h2 transaction-manager-narayana jndi
         karaf@root()> jdbc:ds-create --driverName H2-pool-xa -dbName test test
-        karaf@root()> bundle:install -s mvn:org.jboss.narayana.quickstarts.osgi/osgi-jta-example/5.3.3.Final-SNAPSHOT
+        karaf@root()> bundle:install -s mvn:org.jboss.narayana.quickstarts.osgi/osgi-jta-example/5.3.4.Final-SNAPSHOT
 * Run the commit example
 
         karaf@root()> narayana-quickstart:testCommit
```
Comment (mmusgrov) [inline karaf/osgi-jta/README.md]: Do you need to update this README with the information you added about the admin commands in https://github.com/jbosstm/narayana/pull/1020

Comment (mmusgrov) [inline karaf/osgi-jta/README.md]: @zhfeng  bump

Comment (zhfeng) [inline karaf/osgi-jta/README.md]: ok, I get it and will update the README

Comment (mmusgrov) [inline karaf/osgi-jta/README.md]: Thanks

**Comment by zhfeng** [comment]
@mmusgrov Done

**Pull request merged**